xml/hwbp_registers.xml

```diff
@@ -293,7 +293,7 @@
         <field name="timing" bits="18" access="WARL" reset="0">
             0: The action for this trigger will be taken just before the
             instruction that triggered it is committed, but after all preceding
-            instructions are committed. The exception PC or \RcsrDpc (depending
+            instructions are committed. The \Rxepc or \RcsrDpc (depending
```

Comment:
Just "\Rxepc", not "The \Rxepc".

Same comment 3 more times below.

Reply:
Oops.  Changed.